docroot/themes/humsci/su_humsci_theme/js/hover_menu.js

```diff
@@ -29,25 +29,67 @@
         }
       }
 
+      // Check main menu items to ensure dropdown submenus don't get orphaned or cut off.
+      function menuEdgeCheck() {
+        var viewportWidth = document.body.clientWidth;
```

Comment:
Why not use jquery window width?

Reply:
variable names should start with `$` to be consistent with the others.

Reply:
There's a reason I used clientWidth -- jquery window width includes the scrollbar, I believe.

Updated variable names.

Reply:
https://stackoverflow.com/questions/8339377/how-to-get-screen-width-without-minus-scrollbar

Reply:
can still use jquery

Reply:
@pookmish oh, I see what you're saying... I think I found `document.body.clientWidth` in a thread where someone wrote something like, "you can just use vanilla js for that!"

Reply:
Done.